test/remote/gateways/remote_adyen_test.rb

```diff
@@ -305,6 +305,36 @@ def test_failed_void
     assert_equal 'Original pspReference required for this operation', response.message
   end
 
+  def test_successful_adjust
+    authorize = @gateway.authorize(@amount, @credit_card, @options)
+    assert_success authorize
+
+    assert adjust = @gateway.adjust(200, authorize.authorization)
+    assert_success adjust
+    assert_equal '[adjustAuthorisation-received]', adjust.message
+  end
+
+  def test_successful_adjust_and_capture
+    authorize = @gateway.authorize(@amount, @credit_card, @options)
+    assert_success authorize
+
+    assert adjust = @gateway.adjust(200, authorize.authorization)
+    assert_success adjust
+    assert_equal '[adjustAuthorisation-received]', adjust.message
+
+    assert capture = @gateway.adjust(200, authorize.authorization)
```

Comment:
should this be `@gateway.capture`?

Reply:
Good catch, I think I undo-ed too far after fixing that...

Reply:
(updated)